src/status_im/ethereum/stateofus.cljs

```diff
@@ -12,7 +12,9 @@
 
 (def registrars
   {:mainnet "0xDB5ac1a559b02E12F29fC0eC0e37Be8E046DEF49"
-   :testnet "0x11d9F481effd20D76cEE832559bd9Aca25405841"})
+   ;;NOTE: can be enabled for testing builds
```

Comment:
because it is useful for testing, but for user it will put the app in a weird state if some names are registered on testnet. mainet only is what is in the design and testnet was not working anyway, but it is fixed so it works now if you enable it by uncommenting